chapters/annotations.tex

```diff
@@ -8,6 +8,12 @@ \chapter{Annotations}\label{annotations}
 (For replaceable class declarations with a \lstinline[language=grammar]!constraining-clause! also refer to \cref{constraining-clause-annotations}.)
 The specification in this document defines the semantic meaning if a tool implements any of these annotations.
 
+\section{How to Interpret Annotation Definitions}\label{how-to-interpret-annotation-definitions}
+If the annotation itself is described by a \lstinline!record! \emph{class} the annotation can be seen as a modifier for a \lstinline!record! \emph{component} with the same name.
+A default value for a record member will be used if there is no modifier for that record member.
```

Comment:
Also describe meaning of absent default value:
```suggestion
A declaration equation for a component declaration or record member declaration specifies a default to be used when no corresponding annotation is given.
Default behavior can also be specified in the text as an alternative to a declaration equation.
Component declaration annotations always have default behavior, as all annotations are optional.
When there is no declaration equation and no other explanation of default behavior in the text for a record member, an annotation modifier for the record must contain a modifier for that member.

```

Reply:
As indicated I'm not happy with the last element, but will discuss that separately.